fedimint-core/src/net/peers.rs

```diff
@@ -11,7 +11,6 @@ use crate::task::Cancellable;
 #[cfg(not(target_family = "wasm"))]
 pub mod fake;
 
-/// Owned [`PeerConnections`] trait object type
 pub struct PeerConnections<M>(Box<dyn IPeerConnections<M> + Send + 'static>);
```

Comment:
Should we call it `DynPeerConnections` like other dyn types?

Reply:
Yes.